pkg/controller/controller_instance.go

```diff
@@ -333,6 +335,8 @@ func (c *controller) reconcileServiceInstanceAdd(instance *v1beta1.ServiceInstan
 			// over with a fresh view of the instance.
 			return err
 		}
+		// recordStartOfServiceInstanceOperation has updated the instance, so we need to continue in the next iteration
+		return nil
```

Comment:
This is going to open the window more for #1792. I don't suggest that we do anything about the issue for this PR. In the future, we'll want to check the in-progress properties to make sure that they haven't changed as well.